app/services/solidus_bolt/accounts/detail_service.rb

```diff
@@ -33,6 +35,13 @@ def build_options
           }.merge(authentication_header)
         }
       end
+
+      def update_user(response)
+        user = Spree::User.find_by(email: response['profile']['email'])
+        return unless user
+
+        SolidusBolt::Users::AddPaymentSourceService.call(user: user, payment_methods: response['payment_methods'])
+      end
```

Comment:
This service should just be used to retrieve bolt account info.

Reply:
removed